plugins/dualGovernance/hooks/useCanCreateProposal.tsx

```diff
@@ -56,29 +80,14 @@ export function useCanCreateProposal() {
     contractReads?.[2]?.status,
   ]);
 
+  useEffect(() => {
+    setHasCreatePermission(!!hasCreatePermissionData);
+  }, [hasCreatePermissionStatus]);
+
   // Check if PROPOSER_PERMISSION is granted to the current wallet
   useEffect(() => {
     if (!address || !daoAddress) return;
-
-    readContract(config, {
-      chainId: PUB_CHAIN.id,
-      address: daoAddress,
-      abi: DaoAbi,
-      functionName: "hasPermission",
-      // where, who, permissionId, data
-      args: [
-        PUB_DUAL_GOVERNANCE_PLUGIN_ADDRESS,
-        address,
-        keccak256(toHex("PROPOSER_PERMISSION")),
-        "0x",
-      ],
-    })
-      .then((result) => {
-        setHasCreatePermission(!!result);
-      })
-      .catch((err) => {
-        console.error(err);
-      });
+    hasCreatePermissionRefetch();
```

Comment:
See comment about enabling/disabling query

Reply:
This hook is to update the values in case the wallet address changes